pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -295,6 +296,33 @@ func (opt *Options) Run(args []string) error {
 		}
 	}
 
+	// Monitor operator metrics
+	operatorMetricsTc := &testCase{
```

Comment:
This doesn't belong here.  You can't take dependencies on extended from this package, so that at a minimum needs to be removed.

You should just be a normal e2e test that runs during the test suite.  You don't need to run at the end (odds are you get run near the middle, which is plenty of time to catch this issue).